latest/index.bs

```diff
@@ -175,7 +175,309 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
                 └── n
 </pre>
 
+Tables {#table-layout}
+----------------------
+The following describes the expected layout for tabular data.
+OME-NGFF tables are compatible with the [AnnData model](https://github.com/scverse/anndata).
 
+<pre>
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images.
+│
+└── 123.zarr
+    |
+    ├── .zgroup
+    |
+    ├── .zattrs
+    |
+    └── tables                # The tables group is a container which holds one or multiple tables that are compatible with AnnData.
+        |
+        │                     # The tables group MAY be in the root of the zarr file.
+        ├── .zgroup           # The tables group MAY be in root or in another group.
+        |
+        ├── .zattrs           # `.zattrs` MUST contain "tables", which lists the keys of the subgroups that are tables. In this case, the only table is "my_table".
+                              # hence `.zattrs` should be equal to `{ "tables": [ "my_table" ] }`.
+        |
+        └── my_table
+            │                     # The table group MAY be in the root of the zarr file.
+            ├── .zgroup           # The table group MAY be in root or in another group.
+            |
+            ├── .zattrs           # `.zattrs` MUST contain "type", which is set to `"ngff:region_table"`
+            |                     # `.zattrs` MUST contain "region", which is the path to the data the table is annotating.
+            |                     # "region" MUST be a single path (single region) or an array of paths (multiple regions).
+            |                     # "region" paths MUST be objects with a key "path" and the path value MUST be a string.
+            |                     # `.zattrs` MUST contain "region_key" if "region" is an array. "region_key" is the key in `obs` denoting which region a given row corresponds to.
```

Comment:
All these `MUST` rules apply to regions table, but don't apply to the use-case of storing points/tracking data in tables as in https://github.com/ome/napari-ome-zarr/pull/81
Presumably we do want to allow that use case (and others)? So these rules could be `MAY` instead of `MUST`?